server/README_developer.md

```diff
@@ -0,0 +1,22 @@
+# Tips for developers
+
+## Start service
+### On local machine
+To start the container on a local machine, go to `lomas/server/` and run `docker compose up`. 
+If you encounter any issue, you might want to run `docker compose down` first.
+
+### On a kubernetes cluser
+-- TODO: can someone fill it: like as few lines as possible
```

Comment:
I can do it with a simple helm install explanation now that we have a helm chart published its quite simple.